config/jobs/kubernetes-sigs/cli-utils/cli-utils-presubmit-master.yaml

```diff
@@ -82,10 +82,10 @@ presubmits:
         resources:
           requests:
             memory: "8000Mi"
-            cpu: 8000m
+            cpu: 7
```

Comment:
i dont actually know if there's a difference between 7000m and 7 in the way k8s handles resource request/limits, but I think using -m is less ambiguous as to meaning (shares vs whole cores).

Reply:
done